include/swift/SILOptimizer/Analysis/ARCAnalysis.h

```diff
@@ -186,8 +186,12 @@ class ConsumedArgToEpilogueReleaseMatcher {
   RCIdentityFunctionInfo *RCFI;
   ExitKind Kind;
   llvm::SmallMapVector<SILArgument *, ReleaseList, 8> ArgInstMap;
+  /// TODO: Replace this with ThisBlock. 
   bool HasBlock = false;
 
+  /// Eventually this will be used in place of HasBlock.
+  SILBasicBlock *ThisBlock;
```

Comment:
You seem to already be using this. Is the comment out of date? It's preferred to not commit dead code (even commented), but instead introduce something when it's actually used. Is it possible to remove either ThisBlock or HasBlock and remove the other?

Also, a name more informative than ThisBlock would be helpful here.